frontend/src/components/Schedules.vue

```diff
@@ -96,23 +96,28 @@ const sortedArr = computed(() => {
 
 <template>
   <div>
-    <div class="max-w-screen-md mx-auto">
-      <div class="mb-10 flex-col justify-center">
-        <div class="text-center bg-gray-200 bg-opacity-40 pl-10 pr-10 pb-3 pt-3 mb-4" v-if="schedules.length === 0" >
+    <div class="max-w-screen-lg mx-auto">
+      <div class="mb-10">
+        <div class="max-w-screen-lg mx-auto pb-5">
+          <h4 class="md:text-center">Choose time and place to book seats</h4>
+        </div>
+        <div class="text-center bg-gray-700 bg-opacity-50 pl-10 pr-10 pb-3 pt-3 mb-4" v-if="schedules.length === 0" >
           <p>No schedules available</p>
         </div>
-        <div v-for="(cinemaSchedules, cinemaName) in sortedArr" :key="cinemaName" class="bg-gray-200 bg-opacity-40 pl-10 pr-10 pb-3 pt-3 mb-4">
-          <div>
-            <p class="text-black text-2xl uppercase font-semibold pb-2">{{ cinemaName }}</p>
-            <div class="flex flex-wrap justify-between items-center gap-2">
-              <div @click="displaySeat(schedule)" v-for="(schedule, index) in cinemaSchedules" :key="index"
-                   :class="['rounded-3xl','w-[45%]','flex','p-4','uppercase','flex','flex-col','items-center','gap-1',(schedule.cinemaHall.nrOfSeats - bookedSeatsMap.get(schedule.id)) === 0 ? 'bg-secondary' : 'bg-black']">
-                <p class="text-4xl font-semibold">{{ formatDate(schedule.localDateTime) }}</p>
-                <p class="text-xl uppercase ">KL {{ formatTime(schedule.localDateTime) }}</p>
-                <p class="text-sm uppercase">seats:</p>
-                <p class="text-4xl font-semibold">
-                  {{ schedule.cinemaHall.nrOfSeats - (bookedSeatsMap.get(schedule.id)) === 0 ? 'FULL' :  `${schedule.cinemaHall.nrOfSeats - (bookedSeatsMap.get(schedule.id) || 0)}/${schedule.cinemaHall.nrOfSeats }` }}
-                  </p>
+        <div class="flex flex-col md:flex-row justify-center gap-4">
+          <div v-for="(cinemaSchedules, cinemaName) in sortedArr" :key="cinemaName" class="bg-gray-700 bg-opacity-60 px-5 py-3 mb-4">
+            <div>
+              <h4 class="text-2xl md:text-3xl lg:text-4xl uppercase pb-3 md:pb-4 lg:pb-5 xl:pb-6">Cinema: {{ cinemaName }}</h4>
+              <div class="flex flex-wrap justify-start items-center gap-3">
```

Comment:
This will prevent uneven gap on the right in mobile view and also will not center a lonely item. 

```suggestion
             <div class="flex flex-wrap justify-between gap-3">
```